### PR TITLE
Feature/stats

### PR DESCRIPTION
**Pull request by radeklos**
New salt state for fm stats and vagrant for local testing.

### CODE REVIEW

**Comment by radeklos** [comment]
@krak3n CR please.

File: master.d/salt-api.conf [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# Salt API Config
+#
+
+rest_cherrypy:
+  port: 9000
+  address: 0.0.0.0
+  debug: False
+  disable_ssl: True
+
+# vim: set ft=yaml ts=2 sw=2:
```
Comment (krak3n) [inline master.d/salt-api.conf]: Don't think you need this file since we don't run the salt-api on SFM.

File: master.d/vagrant.master.conf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  # Environment
+  # - etcd: master_etcd root=/pillar/%(env)s
+
+external_auth:
+  pam:
+    saltapi:
+      - .*
+      - '@jobs'
+      - '@runner'
+      - '@wheel'
```
Comment (krak3n) [inline master.d/vagrant.master.conf]: You can also remove this since we don't run the Salt API

File: master.d/vagrant.master.conf [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+log_level: debug
+
+master_etcd:
+  etcd.host: 192.168.37.10
+  etcd.port: 4001
+
+local_etcd:
+  etcd.host: 127.0.0.1
+  etcd.port: 4001
```
Comment (krak3n) [inline master.d/vagrant.master.conf]: This can go as well

File: master.d/vagrant.master.conf [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+# Mimics the actual one in production, note this is YAML formatted.
+#
+
+auto_accept: True
+
+log_level: debug
+
+master_etcd:
+  etcd.host: 192.168.37.10
+  etcd.port: 4001
```
Comment (krak3n) [inline master.d/vagrant.master.conf]: This is also not needed

File: states/docker/init.sls [hunk 5]
```diff
@@ -33,6 +33,6 @@ include:
   pip.installed:
     - name: docker-py
     - reload_modules: True
-    - requie:
+    - require:
```
Comment (krak3n) [inline states/docker/init.sls:36]: how did this ever work lol

**Comment by krak3n** [comment]
Looks great :+1:

**Pull request merged**